src/components/Filters/DropdownRangeSlider.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState } from 'react'
+import { composeClasses } from 'lib/classes'
+import FilterInput from './FilterInput'
+import { FilterRangeSlider } from 'components/Filters'
+
+export interface DropdownRangeSliderProps {
+  /** Maximum value for the range */
+  max: number
+  /** Minimum value for the range */
+  min: number
+  /** Initial maximum value */
+  initMaxValue?: number
+  /** Initial minimum value */
+  initMinValue?: number
+  /** Text for the cancel button */
+  cancelText: string
+  /** Unit name to display next to the values */
+  unitName?: string
+  /** Text for the confirm button */
+  confirmText: string
+  /** Function to call when the form is submitted */
+  onSubmit: (value: RangeValues) => void
+  /** Function to call when the dropdown is closed */
+  onClose?: () => void
+  /** Title of the dropdown */
+  title?: string
+  /** Alignment of the dropdown */
+  align?: 'left' | 'right'
+}
+
+type RangeValues = { max: number; min: number }
+
+export const DropdownRangeSlider = ({
+  max,
+  min,
+  initMaxValue,
+  initMinValue,
+  unitName = '',
+  cancelText,
+  confirmText,
+  onSubmit,
+  onClose,
+  title,
+  align = 'left'
+}: DropdownRangeSliderProps) => {
+  const initialState = {
```

Comment:
Review whether it is feasible to use useMemo, to calculate initialState only when relevant dependencies change.